docs/4.0/extend/icons.md

```diff
@@ -10,6 +10,7 @@ Bootstrap doesn't include an icon library by default, but we have a handful of r
 ## Preferred
 We've tested and used these icon sets ourselves.
 
+- [Feather](https://feathericons.com)
```

Comment:
Oh snap I'm an idiot haha. Can do! Hoping to add more info to the preferred ones soon so I can table this maybe.